pgbot/routine.py

```diff
@@ -77,6 +77,14 @@ async def handle_reminders(reminder_obj: db.DiscordDB):
         reminder_obj.write(new_reminders)
 
 
+async def handle_mutes(mute_obj: db.DiscordDB):
+    bot_mutes = mute_obj.get({})
+    for user_id, mute_time in bot_mutes.items():
+        if datetime.datetime.utcnow() >= mute_time:
+            del bot_mutes[user_id]
```

Comment:
hmm, deleting an element during iteration over the dict is gonna error, you'd need to do something like `tuple(bot_mutes.items())` rather than plain `bot_mutes.items()`